### ISSUE TITLE
Why don' channel_selection layer should stop 

### ISSUE THREAD

**Issue by yzlyty** [Eric-mingjie/network-slimming#45]

**Comment by Eric-mingjie** [comment]
It would work also. But I think even if we does not set `requires_grad=False`, then the gradient would still be None. Therefore it does not get updated during back-propogation.
